daprdocs/content/en/concepts/faq/faq.md

```diff
@@ -27,11 +27,27 @@ Creating a new actor follows a local call like `http://localhost:3500/v1.0/actor
 
 The Dapr runtime SDKs have language-specific actor frameworks. For example, the .NET SDK has C# actors. The goal is for all the Dapr language SDKs to have an actor framework. Currently .NET, Java, Go and Python SDK have actor frameworks.
 
-### Does Dapr have any SDKs I can use if I want to work with a particular programming language or framework?
+### How do I know if my Actor Reminders were successfully migrated to the Scheduler control plane service?
+
+Upon upgrading to Dapr v1.15, the Actor Reminders are automatically migrated to the Scheduler control plane service with _no_ loss of reminders. To confirm that the migration was successful, check the Dapr sidecar logs for the following:
```

Comment:
It should also be noted that each replica will only migrate the reminders whose actor type and id are associated with that host. This means that only when all replicas implementing an actor type are upgraded to 1.15, will have all reminders associated with that type be upgraded.